src/components/userprofile.tsx

```diff
@@ -0,0 +1,63 @@
+import { Card, CardContent } from '@/components/ui/card';
+import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar';
+import { Button } from '@/components/ui/button';
+import { Mail, CalendarDays,ChevronLeft,Search,MoreHorizontal } from 'lucide-react';
+import type { UserProfileProps } from '@/types/profile.ts';
+
+export const UserProfile: React.FC<UserProfileProps> = ({ user }) => {
```

Comment:
コンポーネントの宣言の方法は統一したいです。よって
```suggestion
export const UserProfile = ({ user }) => {
```
こう書いてください